sidebars.ts

```diff
@@ -290,7 +290,7 @@ module.exports = {
             },
             {
               type: 'category',
-              label: 'HTTP Sources for Logs, Metrics, Traces, OTLP',
+              label: 'HTTP Sources for Logs, Metrics, Traces, OTLP, Zoom',
```

Comment:
Do we expect this section to grow - will more sources be added besides Zoom? If so, this title is too long. I'd shorten to "HTTP Sources".

Reply:
We will be having growth not on this section, instead we will have growth on Webhooks sources. So now I have created a different folder named "Webhooks Sources" and Zoom source is kept under this. And since we only have one app under this, no index page is created.

Reply:
Thanks for confirming. Looks like you still need to drop the zoom.md file under your newly created "webhook-sources" folder. Right now it's still under the main "hosted-collector" folder.

Reply:
I have created a separate folder called "Webhooks Sources" and zoom.md file is moved under that.